server/src/services/api.service.ts

```diff
@@ -69,13 +75,14 @@ export class ApiService {
         return next();
       }
 
+      const config = await this.configCore.getConfig({ withCache: true });
```

Comment:
The code that loads the config should be inside of the getMetadataTags method in the shared link service.

Reply:
implemented